website/docs/r/msk_cluster.html.markdown

```diff
@@ -187,11 +187,16 @@ The following arguments are supported:
 ### client_authentication Argument Reference
 
 * `tls` - (Optional) Configuration block for specifying TLS client authentication. See below.
+* `sasl` - (Optional) Configuration block for specifying Sasl client authentication. See below.
 
 #### client_authentication tls Argument Reference
 
 * `certificate_authority_arns` - (Optional) List of ACM Certificate Authority Amazon Resource Names (ARNs).
 
+#### client_authentication sasl Argument Reference
```

Comment:
Nit: if you don't mind moving this above the `tls` docs above to keep things in alphabetical order